lib/protocol/networks.js

```diff
@@ -120,7 +120,9 @@ main.checkpointMap = {
   510000: b('040e6b1f2f4cb198a5780d366bf81e591de257642b9267030000000000000000'),
   525000: b('c994fba2bf168333fd969bcfa64f03ca1b62074f9a8f1b010000000000000000'),
   // Monolith Activation
-  530359: b('0391c40195cf8ae3436f3955f1a8444f07468fd08bda1a010000000000000000')
+  530359: b('0391c40195cf8ae3436f3955f1a8444f07468fd08bda1a010000000000000000'),
+  // Nov 15 2018 Upgrade fork:
```

Comment:
Nit: update to 'Magnetic Activation'

Reply:
er, Anomaly? so `// Nov 15 2018  Magnetic Anomaly Activation` ?

Reply:
OK, or just `Magnetic Anomaly Activation`.